client/post-editor/edit-post-status/index.jsx

```diff
@@ -187,6 +188,7 @@ export class EditPostStatus extends Component {
 				<Revisions
 					revisions={ this.props.post && this.props.post.revisions }
 					adminUrl={ adminUrl }
+					toggleNestedSidebar={ this.props.toggleNestedSidebar }
```

Comment:
(to be read after the comment on `editor-sidebar/util` :) )

> Why do we need to pass as a prop?
The entire "nested sidebar" state machine is controlled by the `PostEditor`'s `nestedSidebar` (React) state value, which means we can't really expose a `toggleNestedSidebar` via Redux connect-ing components that are deeper in the tree ... which means we end up with a couple of components having to propagate either `nestedSidebar` or `toggleNestedSidebar`

> What does nestedSidebar mean?
The other comment introduces the notion

> Is this only used for revisions?
yes

> Can it be toggleRevisions?
We could keep the `nestedSidebar` idea, and expose another function `toggleRevision` that only toggles between the 2 states `NONE` / `REVISIONS`, thus decoupling components like `Revisions`, translating the whole nested sidebar state into a boolean state but I don't really see a whole lot of benefits to it.